fix for git issue 4705

davidmu1 opened the pull request.

opbld33 commented:
Docs Build status updates of commit _[1759de6](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4748/commits/1759de67e670a04d42c825b07613d5822e887af1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/user-post-users.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/gitissue4705/api-reference/beta/api/user-post-users.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/user-post-users?view=graph-rest-beta&branch=pr-en-us-4748) (graph-rest-beta) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201906041839298860-4748%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=BGUun0duNOXv3WPzogpLC%2F85FtVjVbrQF%2BCZmeNgJ14%3D&st=2019-06-04T18%3A37%3A31Z&se=2019-07-05T18%3A42%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

davidmu1 commented:
#sign-off

Lauragra commented on api-reference/beta/api/user-post-users.md:
```diff
@@ -13,6 +13,9 @@ ms.prod: "microsoft-identity-platform"
 The request body contains the user to create. At a minimum, you must specify the required properties for the user. You can optionally specify any other writable properties.
 
+>[!NOTE]
+>If you need to enable the creation of external users, see the [invitation manager](../resources/invitation.md).
```
@davidmu1 , I think the title of the link should be "invitation", and the article topic should be "invitation resource type". Can you pls. verify and update?

davidmu1 replied:
@Lauragra , the title of the article that is being linked to is "invitation manager". I don't understand what you mean by "invitation resource type", can you clarify?

Lauragra replied:
@davidmu1 , yes, I think that the topic you're linking to is mis-titled. It's a resource topic, and the name of the resource is "invitation", so for consistency the topic title should be "invitation resource type". I think we should take a look at the topic, as perhaps it's trying to serve the purpose of both a reference a conceptual topic: https://docs.microsoft.com/en-us/graph/api/resources/invitation?view=graph-rest-1.0. Also, I'm not sure it's clear how this is related to creating external users.

davidmu1 commented:
Renamed the invitation article, changed the link text, and added a sentence saying the external users must be created through an invitation.

#sign-off

davidmu1 commented:
oops, signed off before pushing changes.
